docs/misc/constraints.md

```diff
@@ -37,7 +37,7 @@ string MyString
 
 structure Foo {
  @required
- @nullable
+ @defaultValue("N/A")
  bar: String
```

Comment:
```suggestion
 bar: MyString
```

Nice catch, I think this is meant to be changed as well.

Reply:
Thanks, missed that!